core/src/main/java/hudson/model/ListView.java

```diff
@@ -242,6 +264,13 @@ public boolean hasJobFilterExtensions() {
         return items;
     }
 
+    // Used by getItems
+    private static void checkAddItem(Boolean statusFilter, List<TopLevelItem> items, TopLevelItem item) {
```

Comment:
```suggestion
    private static void addItemMatchingStatus(Boolean statusFilter, List<TopLevelItem> items, TopLevelItem item) {
```

Plus corresponding changes.